templates/admin-on-rest/resource-import.js

```diff
@@ -0,0 +1 @@
+import {{{lc}}} from './resources/{{{lc}}}';
```

Comment:
How can we use `resource-import.js`? When I tried your PR, I created an `Admin` node and I used directly my resource:

```js
import React from 'react';
import { Admin } from 'admin-on-rest';
import { hydraClient } from '@api-platform/admin';
import gallery from './admin/resources/gallery';

export default (props) => {
  return (
      <Admin restClient={hydraClient({entrypoint: 'http://127.0.0.1:8000'})}>
        {gallery}
      </Admin>
  );
};
```

It worked fine, but if I tried to use `resource-import.js`:

```js
import React from 'react';
import { Admin } from 'admin-on-rest';
import { hydraClient } from '@api-platform/admin';
import { gallery } from './admin/resource-import';

export default (props) => {
  return (
      <Admin restClient={hydraClient({entrypoint: 'http://127.0.0.1:8000'})}>
        {gallery}
      </Admin>
  );
};
```

It did not work because `resource-import` did not export resources. In that case, we have two solutions:

1. Export each resource in `resource-import` with the following syntax:

```js
export { default as gallery } from './resources/gallery';
```

2. Export only an array of resources:

```js
export default [
  gallery
];
```

IMO, this solution will be better for DX because we just have to use this array in `Admin` rendering:

```js
import React from 'react';
import { Admin } from 'admin-on-rest';
import { hydraClient } from '@api-platform/admin';
import resources from './admin/resource-import';

export default (props) => {
  return (
      <Admin restClient={hydraClient({entrypoint: 'http://127.0.0.1:8000'})}>
        {resources}
      </Admin>
  );
};
```

Reply:
I like the idea **2** but how to get in template the export of the array ?